### PR TITLE
doc: Update docs for ChuanhuChat v5.0

### PR DESCRIPTION
**Pull request by MZhao-ouo**
ChuanhuChat v5.0 has many updates, and we need to update the readme. Can you please help us check if the translation of the README in your language is appropriate?👀 @bropines @ina111

### CODE REVIEW

**Comment by Keldos-Li** [comment]
If you have time, you can also check whether the string translation in ja_JP.json and ru_RU.json is appropriate (you can pull and run the latest code on main branch to see if the text on the page conforms to semantics, or whether there are characters beyond the container boundary, etc.). Thank you very much! 🥹

**Comment by bropines** [comment]
> ChuanhuChat v5.0 has many updates, and we need to update the readme. Can you please help us check if the translation of the README in your language is appropriate?👀 @bropines @ina111

I checked and corrected ReadMe a little, and also added Russian to presets.py, and of course, with all the changes, I created a PR. As for the interface, there are only issues with the text input fields, but I think this is a minor detail.
image
image


I would also like to ask (so as not to change gitignore) to download the promt template in Russian.
Just rename to .json
[Russian_prompts.txt](https://github.com/GaiZhenbiao/ChuanhuChatGPT/files/12614151/Russian_prompts.txt)

**Comment by MZhao-ouo** [comment]
> > ChuanhuChat v5.0 has many updates, and we need to update the readme. Can you please help us check if the translation of the README in your language is appropriate?👀 @bropines @ina111
> 
> I would also like to ask (so as not to change gitignore) to upload the promt template in Russian. Just rename to .json [Russian_prompts.txt](https://github.com/GaiZhenbiao/ChuanhuChatGPT/files/12614151/Russian_prompts.txt)

Maybe you can submit a request to main directly. It's not a risky change.

**Comment by GaiZhenbiao** [comment]
I added it in https://github.com/GaiZhenbiao/ChuanhuChatGPT/commit/5d96469f54fe08a359caee9f94f30d50bd4a9edd

File: readme/README_ru.md [hunk 1]
```diff
@@ -43,33 +39,83 @@
...
+- **Локальная установка LLM**: одним щелчком разверните свою собственную модель языка большого размера.
+
+### 🤖 Системный промт
+- Установка предпосылок через системное сообщение позволяет эффективно играть роль персонажа;
+- Чуаньху Чат предоставляет набор системных шаблонов, нажмите "Загрузить шаблон системного сообщения", затем выберите необходимый шаблон ниже.
+
+### 💬 Обычный диалог
+- Если ответ не удовлетворяет вас, можно попробовать снова с помощью кнопки "Перегенерировать" или просто удалить этот раунд диалога;
+- Поле ввода поддерживает перенос строки, нажмите <kbd>Shift</kbd> + <kbd>Enter</kbd>, чтобы сделать перенос строки;
+- В поле ввода можно использовать клавиши <kbd>↑</kbd> и <kbd>↓</kbd>, чтобы быстро переключаться в истории отправки;
```
Comment (Keldos-Li) [inline readme/README_ru.md:88]: So shall we use the `< kbd >` tag or the `< code >` tag next?

File: readme/README_ja.md [hunk 2]
```diff
@@ -42,22 +38,84 @@
...
+- api-hostの変更が可能です。
+- カスタムプロキシの設定が可能です。
+- 負荷分散のための複数のapiキーのサポートがあります。
+
+### ⚒️ デプロイに関する情報
+- サーバーへのデプロイ：`config.json`ファイルで`"server_name": "0.0.0.0", "server_port": <あなたのポート番号>,"`を設定します。
+- 共有リンクの取得：`config.json`ファイルで`"share": true,`を設定します。ただし、プログラムが実行されている必要があります。
+- Hugging Faceでの使用：右上のコーナーの「Spaceをコピー」を選択し、それから使用することをおすすめします。これにより、アプリの反応が速くなる場合があります。
 
 ## 使う上でのTips
```
Comment (Keldos-Li) [inline readme/README_ja.md:120]: 这一部分是不是要删掉

File: readme/README_ja.md [hunk 3]
```diff
@@ -42,22 +38,84 @@
...
+
+<sup>新!</sup> 履歴が左側に移動し、使いやすくなりました。また、検索（正規表現対応）、削除、リネームが可能です。
+
+<sup>新!</sup> 大きなモデルによる履歴の自動命名が可能になりました（設定または設定ファイルで有効化が必要）。
+
+<sup>新!</sup> 今では 川虎チャット を PWAアプリケーションとしてインストールすることも可能で、よりネイティブな体験ができます！Chrome/Edge/Safariなどのブラウザをサポート。
+
+<sup>新!</sup> 各プラットフォームに適したアイコンで、見ていても気持ちがいい。
+
+<sup>新!</sup> Finetune（微調整）GPT 3.5に対応！
```
Comment (Keldos-Li) [inline readme/README_ja.md:58]: 上面和下面统一用 新！ 或者 New！吧？

**Review by Keldos-Li — CHANGES_REQUESTED** [review]
我再找人看看

File: readme/README_ja.md (hunk 2 above)
Comment (Keldos-Li) [reply readme/README_ja.md:120]: 额，这里引用的位置有问题，应该是

使う上でのTips 这个段落

https://github.com/GaiZhenbiao/ChuanhuChatGPT/blob/570a39dfcef05380ace6015c03e5ad9c5281e026/readme/README_ja.md?plain=1#L120-L129

**Pull request merged**